src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/naming/patternvariablename/Example1.java

```diff
@@ -0,0 +1,23 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="PatternVariableName"/>
+  </module>
+</module>
+*/
+
+//non-compiled with javac: Compilable with Java17
+package com.puppycrawl.tools.checkstyle.checks.naming.patternvariablename;
+
+// xdoc section -- start
+class Example1 {
+  Example1(Object o1){
```

Comment:
`foo(Object o1){`

to minimize diff between exmaples.

Reply:
no done, please do

Reply:
`Example1(Object o1){`  is a constructor.

To change to `foo` , I had to change the class name 
and also had to add explicit class Example1 to satisfy the exampleClassName check.

Is the change okay ?

Reply:
Change constructor to method